instagram-app/src/App.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+// import logo from './logo.svg';
+import './App.css';
+
+import SearchBar from './components/SearchBar/SearchBar';
+import PostContainer from './components/PostContainer/PostContainer';
+import CommentSection from './components/CommentSection/CommentSection';
+import dummyData from './dummy-data';
+
+
+// function App() {
+//   return (
+//     <div className="App">
+//       <header className="App-header">
+//         <img src={logo} className="App-logo" alt="logo" />
+//         <p>
+//           Edit <code>src/App.js</code> and save to reload.
+//         </p>
+//         <a
+//           className="App-link"
+//           href="https://reactjs.org"
+//           target="_blank"
+//           rel="noopener noreferrer"
+//         >
+//           Learn React
+//         </a>
+//       </header>
+//     </div>
+//   );
+// }
+
+class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      post: dummyData
+    };
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <SearchBar />
+        <PostContainer post={this.state.post} />
```

Comment:
This works fine, however, it is conventional to plural names for arrays since most of the time they have more than one element. In this case, it an array of objects.